cmake/SetupCharmAlgorithm.py

```diff
@@ -15,6 +15,7 @@ def create_interface_file(args):
              "// See LICENSE.txt for details.\n\n" \
```

Comment:
commit message:  "in the order that the PDALs"

Add some documentation to the Parallelization section about tensor product and compile times